main.py

```diff
@@ -0,0 +1,24 @@
+from typing import List, Union
+
+_APACHE_TYPES = ["text/plain", "text/plain; charset=ISO-8859-1", 
+                "text/plain; charset=iso-8859-1", 
+                "text/plain; charset=UTF-8"]
+
+check_for_apache = False
+
+#handling resource metadata
+def _get_resource_metadata(content_types, http_origin):
+    if http_origin:
+        if content_types[-1] in _APACHE_TYPES:
+            return True
+
+    return False
```

Comment:
The main purpose of metadata is only for setting `check-for-apache-flag`. So, now I m returning only that. 

`_APACHE_TYPES` can contain byte sequences so should I add particular bytes in that or in a separate variable?

Reply:
Separate variable, for sure.